source/concurrent/Promise.ooc

```diff
@@ -33,6 +33,13 @@ Promise: abstract class {
 	start: static func (action: Func) -> This {
 		_ThreadPromise new(action)
 	}
+	createEmpty: static func -> This { _EmptyPromise new() }
```

Comment:
I thought we decided not to use only `empty` due to the ambiguity between `empty: Bool` `empty: func` `empty: static func -> This`